homeassistant/components/plugwise/entity.py

```diff
@@ -33,28 +33,28 @@ def __init__(
         if entry := self.coordinator.config_entry:
             configuration_url = f"http://{entry.data[CONF_HOST]}"
 
-        data = coordinator.data.devices[device_id]
+        self.data = coordinator.data.devices[device_id]
```

Comment:
This data isn't needed locally

```suggestion
        data = coordinator.data.devices[device_id]
```